### DISCUSSION TITLE
Create a Logo for cli-surf Project

### DISCUSSION THREAD

**Discussion by K-dash** [ryansurf/cli-surf#129] [Ideas]
Hi @ryansurf

How about creating a logo for the cli-surf project? We can easily make one using the following website:

https://logo.com/logos/open-source-software

(I created a logo for [my project](https://github.com/K-dash/flake8-import-guard) using this site, and it turned out quite nice!)

If you're interested, please give it a try and let's see what we can come up with🥳

**Comment by ryansurf** [comment]
@K-dash 

Interesting, your logo looks great. I say we give it a go, I'll see what I can make and send it to you if I like it!

Nice project, by the way. I'll see if I can make any contributions when I have some time 😄

**Comment by K-dash** [comment]
@ryansurf 
Great!! I like your logo very much 🥳
